common/tier4_autoware_utils/include/tier4_autoware_utils/vehicle/vehicle_state_checker.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Tier IV, Inc.
```

Comment:
[nits]
```suggestion
// Copyright 2022 TIER IV, Inc.
```

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/896/commits/bebacf5cb5b231c56a9c327b250ac410ce534b14